firmware/source/core_xy.cpp

```diff
@@ -0,0 +1,90 @@
+#include <numbers>
+#include "peripherals.hpp"
+#include "main.hpp"
+#include "config.hpp"
+#include "core_xy.hpp"
+
+
+namespace cube_hw {
+
+constexpr float PI = std::numbers::pi_v<float>;
```

Comment:
if these constants are just local, I would move them to anonymous namespace
another option: create `firmware/include/constants.hpp` for all various constants

Reply:
moved to anonymous namespace